src/char/barbarian.py

```diff
@@ -153,7 +189,16 @@ def kill_nihlathak(self, end_nodes: list[int]) -> bool:
         self._move_and_attack((-30, -15), self._char_config["atk_len_nihlathak"] * 0.4)
         wait(0.1, 0.15)
         self._cast_war_cry(1.2)
-        self._do_hork(5)
+        wait(0.5, 0.55)
+        #switch to off hand before horking
+        Logger.info('\033[92m'+"SWITCHING TO OFF HAND!!!!!"+'\033[0m')
+        keyboard.send(self._char_config["weapon_switch"])
+        wait(0.1, 0.15)
+        self._do_hork(4)
+        wait(0.5, 0.55)
+        #switch back to main hand
+        Logger.info('\033[92m'+"SWITCHING BACK TO MAIN HAND!!!!!"+'\033[0m')
+        keyboard.send(self._char_config["weapon_switch"])
```

Comment:
should encapsulate weapon switching in the char. E.g. have char.switch_weapon() method that deals with resiliency  of the process. CTA routine should use that one too.